tests/test_app.py

```diff
@@ -1,16 +1,17 @@
 from unittest import TestCase
+from unittest.mock import patch
 import json
 import sys
 
 from werkzeug.datastructures import Headers
 
 import env
 import test_secrets
-sys.modules['api.secrets'] = test_secrets
 import api
 from api.routes import get_hmac
 
 
+@patch.object(api.secrets, 'TOKENS', test_secrets.TOKENS)
```

Comment:
We have to use `patch.object` because `TOKENS` is a list, not a class or method (which are the sorts of objects that `patch` is used for).

Reply:
but will this fix the issue i was addressing? i'm not trying to patch the `TOKENS` object, but rather the entire `api.secrets` module, so that `import api.secrets` doesn't fail when there's no `secrets.py` file.